.circleci/config.yml

```diff
@@ -92,13 +92,11 @@ workflows:
           filters:
             tags:
               only: /.*/
-      # Create a branch called regression and set it to dev to force a
-      # regression test run
       - regression-test:
           filters:
-            branches:
-              only:
-                - regression
+            tags:
+              only: /.*/
```

Comment:
question : what does mean `only : /.*/` ?
Would it be simpler and equivalent to not have this line ?